interfaces/core_test.go

```diff
@@ -29,16 +29,28 @@ import (
 	"github.com/snapcore/snapd/interfaces/ifacetest"
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/snap/snaptest"
+	"github.com/snapcore/snapd/testutil"
 )
 
 func Test(t *testing.T) {
 	TestingT(t)
 }
 
-type CoreSuite struct{}
+type CoreSuite struct {
+	testutil.BaseTest
+}
 
 var _ = Suite(&CoreSuite{})
 
+func (s *CoreSuite) SetUpTest(c *C) {
+	s.BaseTest.SetUpTest(c)
+	s.BaseTest.AddCleanup(snap.MockSanitizePlugsSlots(func(snapInfo *snap.Info) {}))
+}
+
+func (s *CoreSuite) TearDownTest(c *C) {
```

Comment:
Would this run if you had not defined TearDownTest at all? Since we embed BaseTest does it mean this method is implicitly defined because it's in BaseTest already

Reply:
Yes, you're right, it would be called anyway. Since it was merged aready I'll leave it as is, might make it easier anyway if we ever need to add more teardown code.